src/webgpu/api/validation/render_pipeline/fragment_state.spec.ts

```diff
@@ -382,6 +382,15 @@ g.test('pipeline_output_targets,blend')
         ...u.combine('alphaSrcFactor', kBlendFactors),
         ...u.combine('alphaDstFactor', kBlendFactors),
       ] as const)
+      .expand('writeMask', function* (p) {
+        yield 0;
+        for (let i = 1; i <= p.componentCount; i++) {
+          yield (1 << i) - 1;
```

Comment:
IIUC this is 0b1, 0b11, 0b111, 0b1111
But we're interested in the difference between 0 and non-0 so maybe it makes more sense to test values that are "closer" to 0:
0b1, 0b10, 0b100, 0b1000

in particular 0b1000 is ALPHA-only, which could plausibly be incorrectly handled.

Reply:
Fixed as suggested